bdb/file.c

```diff
@@ -7753,9 +7753,11 @@ static int bdb_process_unused_files(bdb_state_type *bdb_state, tran_type *tran,
                 rc = bdb_get_file_version_data(bdb_state, tran, i, &version_num,
                                                bdberr);
             }
-            if (rc == 0 && version_num == file_version) {
-                found_in_llmeta = 1;
-                break;
+            if (rc == 0) {
```

Comment:
Why?

Reply:
Putting back to how old code was...nothing fancy here, just correcting my mistake. Thanks.

Reply:
Thanks!